collectors/cronjob.go

```diff
@@ -70,21 +70,21 @@ var (
 	)
 )
 
-type CronJobLister func() ([]v2batch.CronJob, error)
+type CronJobLister func() ([]batchv1beta1.CronJob, error)
 
-func (l CronJobLister) List() ([]v2batch.CronJob, error) {
+func (l CronJobLister) List() ([]batchv1beta1.CronJob, error) {
 	return l()
 }
 
 func RegisterCronJobCollector(registry prometheus.Registerer, kubeClient kubernetes.Interface, namespace string) {
 	client := kubeClient.BatchV2alpha1().RESTClient()
```

Comment:
Shouldn't that become `BatchV1beta1`?

Reply:
@cpointner you are right! Do you mind creating a PR for this?

Reply:
#311 (CLA is in the works)

Reply:
Thanks a lot! Feel free to ping us once you're ready with the CLA!